mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -131,10 +131,12 @@ private[regression] trait GeneralizedLinearRegressionBase extends PredictorParam
  * It supports "gaussian", "binomial", "poisson" and "gamma" as family.
  * Valid link functions for each family is listed below. The first link function of each family
  * is the default one.
+ * {{{
  *  - "gaussian" -> "identity", "log", "inverse"
  *  - "binomial" -> "logit", "probit", "cloglog"
  *  - "poisson"  -> "log", "identity", "sqrt"
  *  - "gamma"    -> "inverse", "identity", "log"
+ * }}}
```

Comment:
It's OK but we can just write this better:

- "gaussian": "identity", "log", "inverse"

There's no need for "->" in any docs to mean "to"